index.html

```diff
@@ -12,10 +12,71 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="header"> <!-- contains logo and nav -->
+        <div class="container">   
+            <div class="header__inner">
+               <img src="img/karma-logo.svg" width="25" alt="karma-logo">
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+                <nav class="nav">
+                    <a  href="#">Meet Karma</a>
+                    <a  href="#">How it Works</a>
+                    <a  href="#">Store</a>
+                    <a  href="#">Blog</a>
+                    <a  href="#">Help</a>
+                    <a  href="#">Login</a>
+                </nav>
+            </div>
+        </div>
+    <!-- header starts here-->
+    </header>
+        <div class="hero">
```

Comment:
Why is the hero not inside `main`?
Anything that isn't the header or the footer can go inside main.